api-docs/package.json

```diff
@@ -5,7 +5,10 @@
   "description": "InfluxDB API documentation",
   "license": "MIT",
   "dependencies": {
+    "@redocly/openapi-cli": "^1.0.0-beta.61",
```

Comment:
Where this is still in beta, we'll want to make sure that the generate process fails on error. I vaguely remember disabling exist on error because, at one point, redoc only exited with non-zero exit codes. I don't know if that's still the case. Either way, just something to watch out for.

Reply:
Crossed my mind and forgot to test.

> redoc only exited with non-zero exit codes
I'm not sure I follow. You mean even when there wasn't an error?

I'm testing `set -e`. `openapi` fails and exits when given a bad path or missing reference in the JS. `redoc-cli` fails and exits when given a bad path in options.

Reply:
Yeah, if I remember correctly, success or failure, it would exit with a non-zero code. But this was a looong time ago.